.github/workflows/build_db_clean_image.yml

```diff
@@ -0,0 +1,37 @@
+name: build_db_clean_image
+
+on:
+  schedule:
+    - cron:  '0 6 * * 6'
+
+env:
+  AWS_DEFAULT_REGION: us-west-2
+
+jobs:
+  build_db_clean_image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Login to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9 # v1.10.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3 # v1.5.10
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Build the db cleaner image and push to ECR
+        env:
+          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+        run: |
+          ./scripts/build_db_image "Dockerfile.db_clean" "easi-db-clean" "latest"
```

Comment:
Did you look at https://github.com/aws-actions/amazon-ecr-login to replace the ECR login part of the `build_db_image` script?

Reply:
I did not, but will do so